Dockerfile

```diff
@@ -51,10 +52,10 @@ FROM base-builder as freesurfer-installer
 WORKDIR /opt/freesurfer
 RUN curl -fsSLO https://raw.githubusercontent.com/freesurfer/freesurfer/v7.1.1/distribution/FreeSurferColorLUT.txt
 
-# Download minified FSL (6.0.4-2)
+# Download minified FSL (6.0.7.7)
```

Comment:
Is there a strong motivation to update the FSL version here? We try to avoid introducing anything that would modify results (over and above "bug fixes") within a patch release.

Reply:
I created this PR based on your modifications: https://github.com/MRtrix3/mrtrix3/blob/container_eddyqc_fix/Dockerfile.